R/fb_player_scouting.R

```diff
@@ -39,7 +39,7 @@ fb_player_scouting_report <- function(player_url, pos_versus, time_pause=3) {
   # put sleep in as per new user agreement on FBref
   Sys.sleep(time_pause)
 
-  player_page <- .load_page(player_url)
+  player_page <- xml2::read_html(player_url)
```

Comment:
I reverted page loads for fb_player_scouting_report() back to xml2::read_html(), rather than .page_load() for one reason - I think some users absolutely abuse this function and try to get scouting reports for whole leagues at a time (or the whole of the Big 5) and I want to minimise the risk of these headers also ending up on a blocked list... thoughts @tonyelhabr?

Reply:
I'm ok with this choice 👍 . Unfortunately, it wouldn't make sense for use to make a `load_` function for this, given how the look-back window for the 365-day player scouting reports change daily.

Reply:
Yeah exactly. If our thinking changes on this, we can always implement the .`page_load()` to this function down the track